packages/config/src/options.ts

```diff
@@ -381,11 +381,11 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     overrideLevel: 'any',
   }, {
     name: 'testIsolation',
-    defaultValue: 'on',
+    defaultValue: true,
     validation: (key: string, value: any, opts: ValidationOptions) => {
       const { testingType } = opts
 
-      let configOpts = ['on', 'off']
+      let configOpts = [true, false]
```

Comment:
I feel like this should use `validate.isBoolean`, but for component testing, would need add either `validate.isTrue` or `validate.isExactValue`.

Reply:
Looking at this, `validate.isBoolean` accepts a null value, and we only want true & false. Not sure it's the best option for this?